external/twitter/src/main/scala/org/apache/spark/streaming/twitter/TwitterInputDStream.scala

```diff
@@ -85,9 +93,14 @@ class TwitterReceiver(
         }
       })
 
-      val query = new FilterQuery
-      if (filters.size > 0) {
-        query.track(filters.toArray)
+      if (filters.size > 0 || locations.size > 0) {
```

Comment:
Trivial but can this be `.nonEmpty` instead of `.size > 0` around here? slightly more readable